changelogs/1.23.5.yaml

```diff
@@ -0,0 +1,12 @@
+date: February 27, 2023
```

Comment:
Seems to me that this file and the 1.24.3 file should not be part of the release-1.25 branch, but maybe I'm missing something.

Reply:
yes - they are included as they are historical versions ot this one

see version histories here https://storage.googleapis.com/envoy-pr/0052d73/docs/version_history/version_history.html

Reply:
i tend to sync these separately now to prevent this confusion but in this case the PR was waiting for the other releases so i just synced with the release commit